CQLCookbook.md

```diff
@@ -0,0 +1,349 @@
+# CQL Cookbook
+Reference for creating a CQL prepopulation file for a DRLS Ruleset.
+
+
+## Overview
+As described in the Ruleset Development 101 document, DRLS invovles the prepopulation of clinical questionnaires from a patient's electronic health record (EHR). This prepopulation is done by means of a clinical quality language (CQL) file that fetches FHIR resources form the EHR.
+
+## Contents
+Every ruleset within the CDS-Library repo will contain at least one CQL prepopulation file. These CQL files tend to follow the same generic format. The instructions below will provide:
+1. [CQL Template](#1-cql-template): A generic template to set up your DRLS prepopulation file.
+   - A) Header
+   - B) Codesystems
+   - C) Value Sets
+   - D) Codes
+   - E) Request Parameter
+   - F) Patient Context
+   - G) Define Statement
+      - "Query" Statements
+      - "Function" Statements
+2. [DRLS Statement Templates](#2-drls-statement-templates): A list of frequently used functions that can be copied and modified to be used within your respective prepopulation file. 
+   - I) [Condition Resource Statements](#i-condition-resource-statements)
+      - *List of All Active Conditions*
+      - *List of Patient's 'Other Diagnoses'*
+      - *List of All Relevant Conditions (as specified by a partiular value set)*
```

Comment:
I would switch the order of "Relevant" and "Other" diagnoses.

Reply:
I agree